src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -2190,6 +2190,38 @@ private function addMailerSection(ArrayNodeDefinition $rootNode, callable $enabl
                             ->end()
                         ->end()
                     ->end()
+                    ->children()
+                        ->arrayNode('signer')
```

Comment:
those nodes are missing `->addDefaultsIfNotSet()`, which would avoid the need to repeat defaults in the DI extension